### PR TITLE
Photolithography branch

### PR DESCRIPTION
**Pull request by BLocky104**
This PR adds :

* Machine/multiblocks :

  - Ion Implanter
  - CVD Unit
  - Engraving Unit

* Lines :

    - Doping
    - Helium 4 production and usages
    - White Phosphorus Line

* Miscs :

  
  * change around material colors
  
  * Material Flags for FPM and FPT
  
  * add Naquadria Charge
  
  * recipes for Barium Titanate and Color fixes
  
  * Engraved UHASOC -> Engraved HASOC

### CODE REVIEW

**Comment by BLocky104** [comment]
Reviewed and resolved every problem

**Review by jude123412 — APPROVED** [review]
Looks all good to me 👍

File: src/main/java/gtb/loaders/recipe/Photolithography.java [hunk 1]
```diff
@@ -0,0 +1,1457 @@
...
+import gtb.api.unification.materials.GTBMaterials;
+import gtb.common.item.GTBMetaItems;
+
+public class Photolithography {
+
+    public static void init() {
+        CHEMICAL_RECIPES.recipeBuilder()
+                .fluidInputs(Redstone.getFluid(1000))
+                .input(MetaItems.ULTRA_HIGH_POWER_INTEGRATED_CIRCUIT_WAFER)
+                .output(NANO_PIC_WAFER)
```
Comment (tekcay) [inline src/main/java/gtb/loaders/recipe/Photolithography.java:25]: this recipe and many others are not working, `Recipe output are empty`.

**Review by tekcay — CHANGES_REQUESTED** [review]
Errors loading multiple items texture due to invalid models.
Errors in recipes in Photolithography

**Review by tekcay — COMMENTED** [review]
still missing some :p

**Comment by jude123412** [comment]
And thats why 2 people are needed for changes

**Review by jude123412 — APPROVED** [review]

**Review by jude123412 — APPROVED** [review]

**Comment by tekcay** [comment]
@BLocky104 

- Last recipe in `Photolithography` :
```java
        ASSEMBLER_RECIPES.recipeBuilder()
                .input(CONTROLLED_SHAPED_NAQUADRIA_CHARGE)
                .input(NAQUADRIA_CHARGE)
                .output(CONTROLLED_SHAPED_NAQUADRIA_CHARGE)
                .EUt(250) // TODO was missing, I don't which value you want
                .duration(120)
                .circuitMeta(2)
                .buildAndRegister();
```

the EUt value was missing, just added a arbitrary value.
                
- some textures are missing : 

    - shaped_high_energy_gamma_shield
    - hasoc_lithography_mask
    - naquadria_charge
    - engraved_oganesson_wafer
    - photopolymer_plate

**Review by jude123412 — APPROVED** [review]

**Review by jude123412 — APPROVED** [review]

**Review by tekcay — APPROVED** [review]

**Pull request merged**